packages/cli/Dockerfile

```diff
@@ -15,8 +15,8 @@ RUN npm install sharp@0.30.7
 
 # Install the landing assets
 COPY ./basemaps-landing*.tgz /app/
-RUN npm install ./basemaps-landing*.tgz
+COPY ./basemaps-cogify*.tgz /app/
 
-COPY dist/index.cjs /app/
```

Comment:
This copy bundled index.cjs been removed?

Reply:
oh i missed my last commit :(